Examples/ObjectiveC/AnnotationViewExample.m

```diff
@@ -21,11 +21,14 @@ - (void)layoutSubviews {
 - (void)setSelected:(BOOL)selected animated:(BOOL)animated {
     [super setSelected:selected animated:animated];
 
-    // Animate the border width in/out, creating an iris effect.
-    CABasicAnimation *animation = [CABasicAnimation animationWithKeyPath:@"borderWidth"];
-    animation.duration = 0.1;
-    self.layer.borderWidth = selected ? self.bounds.size.width / 4 : 2;
-    [self.layer addAnimation:animation forKey:@"borderWidth"];
+    // Increase the size of the annotation if it is selected, and restore annotation to its original size if it is not selected.
+    CABasicAnimation *animation = [CABasicAnimation animationWithKeyPath:@"bounds.size"];
+    [self.layer addAnimation:animation forKey:@"bounds.size"];
```

Comment:
In general we should prefer the higher level animation constructs where possible. Let's look at whether something like
```
[UIView animateWithDuration:kSelectionAnimationDuration animations:^{ 
    self.transform = CGAffineTransformMakeScale(2.0, 2.0);
}];
```
would be more appropriate. (We still need to support iOS 9, otherwise I think the new `UIViewPropertyAnimator` would be preferable.)

In addition, in the above you add an animation for `bounds.size` - but when you scale by modifying the transform `bounds` doesn't change size (`frame` is a computed property and would).

Reply:
We probably want to check the `animated` method parameter too - to decide whether the scale should be animated.